docs/lib/datastore/fragments/ios/auth-model.md

```diff
@@ -0,0 +1,104 @@
+<amplify-callout>
+When using the `@auth` directive with DataStore, currently only following fields are supported:
+
+1. `allow` with "owner" and "groups"
+2. `ownerField`
+3. `groups`
+4. `operations`
+
+</amplify-callout>
+
+## Updated schema
+
+Create a Model that is read-only.
+
+```graphql
+type Todo
+    @model
+    @auth(rules: [
+        { allow: owner, ownerField: "owner", operations: [create, update, delete] },
+    ]) {
+    id: ID!
+    content: String!
+    owner: String
+}
+```
+
+The API will allow users to query, list, and subscribe to model creation, updates, deletions by other users. A user can only update and delete their own model, that is a model can not be updated or deleted by another user.
+
+## API with Cognito User Pool auth
+
+With the schema, make sure you have selected Cognito User Pool as the auth
+
+```console
+? Please select from one of the below mentioned services: 
+  `GraphQL`
+? Select from the options below 
+  `Update auth settings`
+? Choose the default authorization type for the API 
+  `Amazon Cognito User Pool`
+```
+
+Once you have updated the schema and selected Cognito User Pool as the authorization type, run `amplify push` 
+
+## Configuration
+
+`amplifyconfiguration.json` will have section under "awsAPIPlugins" and "awsCoognitoAuthPlugin" with the API and Auth configurations
```

Comment:
Again, we should have a lead in here.

Upon successfully running `amplify push` per the last step, `amplifyconfiguration.json` will have a section under "awsAPIPlugins" and "awsCoognitoAuthPlugin" with the corresponding API and Auth configurations.  Your `amplifyconfiguration.json` may look like this: